app/src/main/kotlin/com/wire/android/ui/authentication/create/username/CreateAccountUsernameScreen.kt

```diff
@@ -100,14 +109,16 @@ private fun UsernameContent(
 @Composable
 private fun UsernameTextField(
     state: CreateAccountUsernameViewState,
-    onUsernameChange: (TextFieldValue) -> Unit
+    onUsernameChange: (TextFieldValue) -> Unit,
+    forbiddenCharactersRegex: Regex,
```

Comment:
I think this should be part of ViewModel

Reply:
I wanted this animation to be handled only by UI element, but finally moved this logic to ViewModel.